### PR TITLE
added aria-hidden=true to caret down icon in dropdown component.

### LINKED ISSUE #705: Accessibility: add aria-hidden=true to the dropdown icon
Within the dropdown component, the caret down icon is missing aria-hidden=true. 

Add aria-hidden=true to the icon inside of the dropdown-toggle-icon hbs file.

### PR DESCRIPTION
**Pull request by christiemolloy**
Close #705

Added aria-hidden=true to caret down icon in dropdown component.

As per the[ accessibility docs](https://pf-next.com/accessibility-guide#icons) all icons should have `aria-hidden="true"`. This was discovered during the toolbar demo work. @jgiardino please advise if this is the wrong approach.

Thanks!

### CODE REVIEW

**Comment by christiemolloy** [comment]
the dropdown-top-example.hbs file has been updated: the icon has been removed and replaced with `pf-c-dropdown--IsExpanded="true"` as an attribute @matthewcarleton

**Comment by jgiardino** [comment]
Thanks for updating this!

Since we're adding the aria-hidden attribute to the html example, we should add it to the accessibility documentation, too, with a description like "Hides the icon from assistive technologies" (but feel free to reword this).

In the example below, `true` is surrounded by curly quotes (“”) instead of straight quotes (""). It seems fine, but also weird. I'm assuming should fix this?
`<i class="fas fa-caret-down pf-c-dropdown__toggle-icon" aria-hidden=“true”></i>`

Do we also want to add `aria-hidden="true"` to the kebab icon? Is there a reason to do one and not the other?

File: src/patternfly/components/Dropdown/examples/dropdown-kebab-example.hbs [hunk 1]
```diff
@@ -1 +1 @@
-{{#> dropdown id="dropdown-example-kebab" pf-c-dropdown--IsExpanded="true" pf-c-dropdown--Modifier="pf-m-plain" aria-label="Actions"}}<i class="fas fa-ellipsis-v"></i>{{/dropdown}}
\ No newline at end of file
+{{#> dropdown id="dropdown-example-kebab" pf-c-dropdown--IsExpanded="true" pf-c-dropdown--Modifier="pf-m-plain" aria-label="Actions"}}<i class="fas fa-ellipsis-v"  aria-hidden="true"></i>{{/dropdown}}
```
Comment (matthewcarleton) [inline src/patternfly/components/Dropdown/examples/dropdown-kebab-example.hbs]: I'd say for consistency let's pull the icon out of there and have a `pf-c-dropdown__kebab-icon="true"` and create a file for the icon.

Comment (christiemolloy) [reply src/patternfly/components/Dropdown/examples/dropdown-kebab-example.hbs]: Agreed. The icon has been pulled out and placed in a file. pf-c-dropdown__kebab-icon="true" now pulls in the icon.

**Comment by jgiardino** [comment]
Thanks for making the updates! No more suggestions from me.

**Pull request merged**